orchestrator/redisManager.js

```diff
@@ -0,0 +1,22 @@
+/* jslint node: true */
+"use strict";
+
+var redis = require("redis");
+var config = require("./config");
+var { ClientWrapper } = require('./redisClientWrapper');
+
+class RedisManager {
+  constructor() {
+    this.redis = redis.createClient({ host: config.redis.url });
```

Comment:
how works the reconnection, it's automatically?

Reply:
Yes, it's automatic